backend-project/small_eod/institutions/factories.py

```diff
@@ -0,0 +1,35 @@
+from factory.django import DjangoModelFactory
+import factory.fuzzy
+from .models import AddressData, ExternalIdentifier, Institution
+
+
+class PolishFaker(factory.Faker):
+    def __init__(self, *args, **kwargs):
+        kwargs["locale"] = "PL"
+        super().__init__(*args, **kwargs)
+
+
+class AddressDataFactory(DjangoModelFactory):
+    city = PolishFaker("city")
+    voivodeship = PolishFaker("region")
+    flat_no = PolishFaker("building_number")
+    street = PolishFaker("street_name")
+    postal_code = PolishFaker("postcode")
+    house_no = PolishFaker("building_number")
+    email = PolishFaker("email")
+    epuap = factory.Sequence(lambda n: "epuap-%s" % n)  # todo provide example epuap
```

Comment:
```suggestion
    epuap = factory.Sequence(lambda n: "/epuap-%s/SkrytkaESP" % n)  # todo provide example epuap
```

Kolumna URI w https://epuap.gov.pl/wps/portal/strefa-urzednika/pomoc_urzednik/!ut/p/a1/tZJPT4NAEMW_Ch44kp3lP0dErMU2pUVs4UJGoBVb2C1Q2_TTS200RqOtie5tkzcvv_dmSExmJK7wuVhgW7AKV4d_rCfUklUqB-BBr6-C7Tiuc3PtyUC1ThB9FPTMwWUnGFr3d0NDgRGcNw_fPPvk_JTEJE6rlrePJMr5BrksAl-J0LR1PkdhU-_zrCqWmHBWslSEbIVCUbV5vaixZTXbirBsCtwvUcCszhu2RSFv-MGWp0VGoszQM5OqmmToqSKppm5KVkYNSUFDtlINlTnVD2r20OyxJpEIbxj-QITgiBG-Y_hHjKuvGLefMdzAP-Y70fCr4IcKo65jI5Etud-zTBiA5nYljj3bof4IYExJ8Pu0U2eYjEN30nn_T3DvjMspntbr2O72zzrDXUtmf3kAvAzD0lTKJPDL0txJcWRfvABxA4jv/dl5/d5/L2dBISEvZ0FBIS9nQSEh/?obszar=%2Fepuap2%2FPL%2FStrefa%20Urzednika_Pomoc%2FDla%20intergratorow%2FKsiazka%20adresowa%20ESP .